pandas/tests/frame/test_constructors.py

```diff
@@ -1555,7 +1555,6 @@ def test_constructor_mixed_type_rows(self):
         "tuples,lists",
         [
             ((), []),
-            ((()), []),
```

Comment:
Hm, was this supposed to contain an inner tuple like
``((),)``

Reply:
Ah possibly yes. That passes here so added it